### PR TITLE
roachtest: update version map and fixtures

### PR DESCRIPTION
**Pull request by celiala**
This commit adds the recently released 19.2.11, and 20.1.7
to the version map in PredecessorVersion.

Release note: None (testing change)

### CODE REVIEW

**Comment by celiala** [comment]
For release tracking issues:

* 19.2.11 - https://github.com/cockroachdb/cockroach/issues/55256
* 20.1.7 - https://github.com/cockroachdb/cockroach/issues/55257

File: pkg/cmd/roachtest/test_runner.go [hunk 1]
```diff
@@ -1171,8 +1171,8 @@ func PredecessorVersion(buildVersion version.Version) (string, error) {
 	// (see runVersionUpgrade). The same is true for adding a new key to this
 	// map.
```
Comment (jlinder) [inline pkg/cmd/roachtest/test_runner.go:1172]: The fixtures also need to be updated like in [this PR](https://github.com/cockroachdb/cockroach/pull/50770).

Note, there were some issues with this in the last couple releases that should be fixed now. If you need it, see https://github.com/cockroachdb/cockroach/pull/55253 for reference.

**Comment by celiala** [comment]
Thanks @arulajmani and @irfansharif to helping me figure out how to generate the fixtures! 💯 

This is RFAL, thank you!

File: pkg/cmd/roachtest/test_runner.go [hunk 2]
```diff
@@ -1171,8 +1171,8 @@ func PredecessorVersion(buildVersion version.Version) (string, error) {
...
 	verMap := map[string]string{
-		"20.2": "20.1.6",
-		"20.1": "19.2.9",
+		"20.2": "20.1.7",
+		"20.1": "19.2.11",
```
Comment (irfansharif) [inline pkg/cmd/roachtest/test_runner.go:1175]: I only see the fixtures in `pkg/cmd/roachtest/fixtures/*/checkpoint-v20.1.tgz` being updated, but it seems we're also bumping the version map for 19.2.11? If I've understood correctly, we'll need to generate fixtures for that too. We can do so by changing `newV` below for one run and copying those artifacts in.

File: pkg/cmd/roachtest/versionupgrade.go [hunk 3]
```diff
@@ -94,7 +94,7 @@ func runVersionUpgrade(ctx context.Context, t *test, c *cluster, buildVersion ve
 		// The version to create/update the fixture for. Must be released (i.e.
 		// can download it from the homepage); if that is not the case use the
 		// empty string which uses the local cockroach binary.
-		newV := "20.1.6"
+		newV := "20.1.7"
```
Comment (irfansharif) [inline pkg/cmd/roachtest/versionupgrade.go:97]: Here.

**Review by irfansharif — APPROVED** [review]

File: pkg/cmd/roachtest/test_runner.go (hunk 2 above)
Comment (celiala) [reply pkg/cmd/roachtest/test_runner.go:1175]: Ah, thank you! Done :)

**Comment by celiala** [comment]
Thanks for the reviews and all the help!

**Pull request merged**